.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[alias]
+lambda-deploy = "!sh -c 'make -C crates/embucket-lambda deploy \"$1\"' --"
```

Comment:
This alias values looks really strange, and for sure we don't need two versions of them. One would be enough.